crates/agent/src/integration_tests/mod.rs

```diff
@@ -11,3 +11,81 @@ mod schema_evolution;
 mod source_captures;
 mod unknown_connectors;
 mod user_publications;
+
+// TODO(johnny): Do we need this? It's used in only one integration test.
```

Comment:
Didn't catch this earlier, but I'm changing that test a bunch anyway, and can remove this while I'm at it.